Common/include/option_structure.hpp

```diff
@@ -2225,6 +2225,18 @@ struct StreamwisePeriodicValues {
   su2double Streamwise_Periodic_InletTemperature;   /*!< \brief Area avg static Temp [K] at the periodic inlet. Used for adaptive outlet heatsink. */
 };
 
+/*!
+ * \brief Type of POD basis generation (for use with libROM)
+ */
+enum ENUM_POD_KIND {
+  STATIC_POD = 1,            /*!< \brief Use static SVD for POD basis generation. */
+  INCREMENTAL_POD = 2        /*!< \brief Use incremental SVD for POD basis generation. */
+};
```

Comment:
```suggestion
enum class POD_KIND {
  STATIC,            /*!< \brief Use static SVD for POD basis generation. */
  INCREMENTAL        /*!< \brief Use incremental SVD for POD basis generation. */
};
```
We have made a change recently to use `enum class` when possible to keep up with c++ best practices.
You will need POD_KIND instead of unsigned short in some places and e.g. POD_KIND::STATIC instead of STATIC_POD. Easy changes.